marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/hub-entities/generateSearchOptions.sjs

```diff
@@ -11,13 +11,30 @@ const input =
     }
   }];
 
-const options = hent.dumpSearchOptions(input);
+const params = false;
+const options = hent.dumpSearchOptions(input, params);
 
 [
   test.assertEqual("limit=25", xs.string(fn.head(options.xpath("/*:constraint[@name = 'title']/*:range/*:facet-option/text()"))),
     "To avoid displaying large numbers of values in facets in QuickStart, range constraints default to a max of 25 values"
   ),
   test.assertExists(options.xpath("/*:values[@name = 'Book']/*:range/*:element[@name = 'title']")),
+  test.assertExists(options.xpath("/*:transform-results[@apply = 'empty-snippet']")),
   test.assertNotExists(options.xpath("/*:values[@name = 'Book']/*:range/*:facet-option"),
     "A facet option does not need to be set on the values element")
 ];
+
+const expParams = true;
+const expOtions = hent.dumpSearchOptions(input, expParams);
+[
```

Comment:
Just one tweak needed here - the assertions from above are being lost. To correct that, assign the first batch of assertions to an array - e.g. 

    let assertions = [the first batch of assertions];

Then add the others:

    assertions.concat(more assertions)

I tested this locally and verified that 9 assertions should be returned instead of just 5.